server_management/management/commands/deploy.py

```diff
@@ -290,24 +315,22 @@ def validate_choice(choice):
             },
             {
                 'title': 'Update pip',
-                'command': '{} install -U pip'.format(pip_command),
+                'command': f'{pip_command} install -U pip'
             },
             {
                 'title': 'Install virtualenv',
-                'command': '{} install virtualenv'.format(pip_command),
+                'command': f'{pip_command} install virtualenv',
+            },
+            {
+                'title': 'Install supervisor',
+                'command': f'pip2 install supervisor',
```

Comment:
Should this be `pip3`?

Reply:
No, for some reason it doesn't work with pip3. I can't remember why it was.